src/components/ResponsiveImage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Box, Image } from 'grommet'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledBox = styled(Box)`
+  display: inline;
+`
+
+export const StyledImage = styled(Image)`
+  height: auto;
+  max-width: 100%;
+  ${props => props.border && css`border: 1px solid black;`}
+  ${props => props.maxHeight && css`max-height: ${props.maxHeight};`}
```

Comment:
Why not add the border and max-height to the wrapper div box? The child image should expand as needed with in it.

Reply:
I've found the image is often quick to overflow the parent without some sort of height definition